### PR TITLE
Fixes #1097: Restore access specifiers in TwitterScraper.java

### PR DESCRIPTION
**Pull request by vibhcool**
This PR restores access modifiers modified in #1133 and #1098 using Reflection API as suggested by @SKrPl .

I have:
- [x] There is a corresponding issue for this pull request.
- [x] Mentioned the Issue number in the pull request commit message `Fixes #<number> commit message`
- [x] There is only strictly only one commit per issue.

### For the reviewers
I have:
- [ ] Reviewed this pull request by an authorized contributor.
- [ ] The reviewer is assigned to the pull request.

### CODE REVIEW

**Comment by vibhcool** [comment]
Tested and open for review. 
@SKrPl @kavithaenair @singhpratyush @mariobehling @djmgit @SarishiNoHara

**Comment by singhpratyush** [comment]
@vibhcool: Why are we trying to see reflections when we can simply use the `public` methods? Apart from testing with reflections, are there any other reasons for making them `private`?

What I feel like is that methods can stay `public` and be tested the way they were done before.

**Comment by vibhcool** [comment]
@singhpratyush they were private methods, to apply test on them, i made them public. Since this is not a good practice. So I have applied reflection to restore the changes

File: test/org/loklak/harvester/TwitterScraperTest.java [hunk 1]
```diff
@@ -179,4 +193,71 @@ public String dateFormatChange(String time) {
...
+            return privateMethod.invoke(clazz.newInstance(), args);
+        }
+    }
+
+    public Object executePrivateMethod(
+            Class<?> clazz,
+            String methodName,
+            Class<?>[] parameterTypes,
+            Object ... args
+    ) throws Exception{
```
Comment (kavithaenair) [inline test/org/loklak/harvester/TwitterScraperTest.java]: Please maintain spacing consistency as in line 238. Rest looks fine. Good job 👍

Comment (vibhcool) [inline test/org/loklak/harvester/TwitterScraperTest.java]: @kavithaenair , fixed it, please re-review

Comment (kavithaenair) [inline test/org/loklak/harvester/TwitterScraperTest.java]: Looks good 👍 . Rebase it.

**Comment by SKrPl** [comment]
@vibhcool I am not sure regarding the unused method codacy issue, but can you try using `@SuppressWarnings` annotation with `"unused"` parameter. It may fix the codacy issue.

**Comment by mariobehling** [comment]
Please resolve merge conflicts.

**Comment by vibhcool** [comment]
@mariobehling resolved conflicts

@SKrPl made the suggested changes

**Comment by mariobehling** [comment]
Please resolve merge conflicts.

**Pull request merged**